lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -19,61 +19,47 @@
+                ## Translators: provider_name is the name of a credit provider or university (e.g. State University)
+                ${_("You are now eligible to purchase course credit from {provider_name} for this course. Click <strong>Get Credit</strong> to get started.").format(
+                provider_name=credit_status["provider_name"],
+                )}
+            </p>
+            <div class="purchase_credit">
+                <a class="btn credit-btn purchase-credit-btn" href="${settings.ECOMMERCE_PUBLIC_URL_ROOT}/credit/checkout/${credit_status['course_key']}" target="_blank" data-course-key="${credit_status['course_key']}">${_("Get Credit")}</a>
+            </div>
+        % elif credit_status["request_status"] in [None, "pending"] and not credit_status["error"]:
+                <p class="message-copy credit-msg credit-request-pending-msg">
+                    ## Translators: provider_name is the name of a credit provider or university (e.g. State University)
+                    ${_("Thank you for your payment. To receive course credit, you must now request credit at the {provider_name} website.").format(
+                            provider_name=credit_status["provider_name"],
+                        )
+                    }
+                </p>
+                <button class="btn credit-btn pending-credit-btn" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">${_("Finalize Credit")}</button>
 
-        </p>
-        <div class="purchase_credit">
-            <a class="btn credit-btn purchase-credit-btn" href="${settings.ECOMMERCE_PUBLIC_URL_ROOT}/credit/checkout/${credit_status['course_key']}" target="_blank" data-course-key="${credit_status['course_key']}">${_("Get credit")}</a>
-        </div>
-    % elif credit_status["request_status"] in [None, "pending"] and not credit_status["error"] :
-        % if credit_status["request_status"] == "pending":
-            <p class="message-copy credit-request-pending-msg">
-                ## Translators: link_to_provider_site is a link to an external webpage. The text of the link will be the name of a
-                ## credit provider, such as 'State University' or 'Happy Fun Company'
-                ${_("Thank you. Your credit is processing. Please see {link_to_provider_site} for more information.").format(
-                        link_to_provider_site=provider_link,
+        % elif credit_status["request_status"] == "approved" and not credit_status["error"] :
+            <p class="message-copy credit-msg credit-request-approved-msg">
+                ## Translators: provider_name is the name of a credit provider or university (e.g. State University)
+                ${_("<strong>Congratulations!</strong> {provider_name} has converted your course credit. To see your course credit, click <strong>Access Credit</strong>.").format(
```

Comment:
Talk to @pbaruah, @mikigoyal, and @marcotuts to schedule any changes for the next release. This patch is already in production.